tests/test_connector.py

```diff
@@ -1348,6 +1348,15 @@ async def test___get_ssl_context6(loop: Any) -> None:
     assert conn._get_ssl_context(req) is conn._make_ssl_context(True)
 
 
+async def test___get_ssl_context7(loop: Any) -> None:
```

Comment:
> meaningful names

"meaningful" is a bit vague description, so I will put in my two cents here to bring more clarify and a real example. It will be awesome if tests will have the following naming pattern:

 `test_` + `<function_name>` + `(optional) <case or state or expected_behavior>`